docs/interpreter/spark.md

```diff
@@ -343,6 +343,38 @@ myScalaDataFrame = DataFrame(z.get("myScalaDataFrame"), sqlContext)
   </div>
 </div>
 
+In addition to the programmatic access using `z.get()` shown above, 
+some interpreters also support inline interpolation of such objects
+into command lines by using substitution patterns of the form `{object-key}`.
+The following example shows how the value of an object `put` from Scala is included within a SQL command.
+
+<div class="codetabs">
+  <div data-lang="scala" markdown="1">
+
+{% highlight scala %}
+// Put object from scala
+%spark
+val tableName = ...
+z.put("tableName", tableName)
+
+  </div>
+  <div data-lang="sql" markdown="1">
+
+{% highlight sql %}
+# Inline interpolate into SQL command
+%spark.sql
+select * from {tableName}
+
+  </div>
```

Comment:
in this case, this might not be the best way to document in multiple language - I think this way will result in scala and sql in its own tab within one code block in the generated doc? whereas in this case we might prefer to show them in sequence in explain how to flow from one to another